.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
```

Comment:
Gitter history is not easily searchable. If one person has an issue it's likely they're not the only ones to ever have the same or similar problem. Jupyter is moving away from gitter for this reason.